### DISCUSSION TITLE
[@types/chrome] Missing several properties from declarativeNetRequest.RuleCondition

### DISCUSSION THREAD

**Discussion by nev-sc** [DefinitelyTyped/DefinitelyTyped#57759] [Issues with a @types package]
declarativeNetRequest appears to be missing properties added from Chrome 91 onwards, including the following for RuleCondition
- excludedRequestMethods
- excludedTabIds
- requestMethods
- tabIds

https://developer.chrome.com/docs/extensions/reference/declarativeNetRequest/#type-RuleCondition

[a comment by an automated account is not shown]

**Comment by nicolas377** [comment]
Hey there! I've opened a PR for this: #57778

**Reply by nev-sc** [reply]
Sorry, I've been on leave over the holiday season and didn't spot this on my phone:

should
`tabIds?: number | undefined;`
not be
`tabIds?: number[] | undefined;` ?

Also the description appears to be the same as excludedTabIds

**Reply by nicolas377** [reply]
The description of tabIds does seem to be correct, but I'll open a PR to fix the `number` to `number[]` problem.

**Reply by nev-sc** [reply]
Thanks!

**Reply by nicolas377** [reply]
Just opened the PR, see #58010
